source/dotnet/Library/AdaptiveCards.Rendering.Wpf/AdaptiveCardRenderer.cs

```diff
@@ -168,6 +187,24 @@ public static FrameworkElement RenderAdaptiveCardWrapper(AdaptiveCard card, Adap
             return outerGrid;
         }
 
+        private string GenerateLighterColor(string hexColor)
```

Comment:
Please open an issue to consolidate the color parsing code. This code allows for colors without validating the string.